### PR TITLE
Update the docs for the _APP_SMS_PROVIDER env var

### PR DESCRIPTION
**Pull request by stnguyen90**
## What does this PR do?

To handle special characters like '@' or '/', the user and secret values must be URL encoded.

## Test Plan

image

## Related PRs and Issues

- https://github.com/appwrite/appwrite/pull/5916

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/config/variables.php [hunk 1]
```diff
@@ -422,7 +422,7 @@
         'variables' => [
             [
                 'name' => '_APP_SMS_PROVIDER',
-                'description' => "Provider used for delivering SMS for Phone authentication. Use the following format: 'sms://[USER]:[SECRET]@[PROVIDER]'. \n\nAvailable providers are twilio, text-magic, telesign, msg91, and vonage.",
+                'description' => "Provider used for delivering SMS for Phone authentication. Use the following format: 'sms://[USER]:[SECRET]@[PROVIDER]'.\n\nEnsure `[USER]` and `[SECRET]` are URL encoded if they contain any non-alphanumeric characters.\n\nAvailable providers are twilio, text-magic, telesign, msg91, and vonage.",
```
Comment (stnguyen90) [inline app/config/variables.php:425]: @gewenyu99, maybe I should remove the backticks...anything else?

**Comment by stnguyen90** [comment]
I need to update https://appwrite.io/docs/sms-delivery with instructions on how to URL encode.

**Review by gewenyu99 — APPROVED** [review]
Remember to also update the main docs repo with more instructions. This by itself LGTM. 

No commnets on back ticks.

**Comment by stnguyen90** [comment]
> Remember to also update the main docs repo with more instructions. 

Here's the PR: https://github.com/appwrite/docs/pull/402

**Comment by stnguyen90** [comment]
> Remember to also update the main docs repo with more instructions. 

@gewenyu99, here's the PR: https://github.com/appwrite/docs/pull/402

**Pull request merged**